stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -82,6 +80,17 @@ public open class MessageListItemViewHolderFactory {
         }
     }
 
+    private fun createTextAndAttachmentViewHolder(parentView: ViewGroup): BaseMessageItemViewHolder<out MessageListItem> {
```

Comment:
Should we make this `protected` so that clients can call it, like the other similar methods here?

Reply:
Fixed